util/math/math.go

```diff
@@ -48,3 +48,14 @@ func StrLenOfInt64Fast(x int64) int {
 	}
 	return size + StrLenOfUint64Fast(uint64(Abs(x)))
 }
+
+// MinInt choice smallest integer from its arguments.
+func MinInt(x int, xs ...int) int {
```

Comment:
why not define this func as `MinInt(x ...int) int`?

Reply:
just copied from br/utils. I guess in this way we could handle empty parameters at compile time

Reply:
`mathutil.MinVal()` can be used